plugin/cog/cog-rangereader-azure/pom.xml

```diff
@@ -0,0 +1,60 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
This pom.xml does not seem to be included in the parent pom, nor in the cog-all profile... for reference, here are all places where the ``cog-rangereader-http`` is mentioned:

```
aaime@colossus ~/devel/imageio-ext ((9b439f2e...)) $ git grep cog-rangereader-http
plugin/cog/README.md:* HTTP: [HttpRangeReader](./cog-rangereader-http/src/main/java/it/geosolutions/imageioimpl/plugins/cog/HttpRangeReader.java)
plugin/cog/cog-rangereader-http/pom.xml:    <artifactId>imageio-ext-cog-rangereader-http</artifactId>
plugin/cog/cog-reader/pom.xml:                    <artifactId>imageio-ext-cog-rangereader-http</artifactId>
plugin/cog/cog-reader/pom.xml:                    <artifactId>imageio-ext-cog-rangereader-http</artifactId>
plugin/cog/cog-reader/pom.xml:                    <artifactId>imageio-ext-cog-rangereader-http</artifactId>
plugin/cog/pom.xml:        <module>cog-rangereader-http</module>
plugin/cog/pom.xml:                <artifactId>imageio-ext-cog-rangereader-http</artifactId>
```

Reply:
Fixed